changelogs/fragments/T6829-ospfv3-v1.3-area-interfacces.yml

```diff
@@ -0,0 +1,6 @@
+---
+bugfixes:
+  - vyos_ospfv3 - added interface(s) new parameter to the area attribute (both facts and config) for 1.3- and 1.4+
```

Comment:
Recommend tightening this:
```
- vyos_ospfv3 - added support for adding interfaces to areas 
```

This is more direct on the value, and since we're supporting all supported versions, the version information is unnecessary.